spec/custom/index.html

```diff
@@ -176,7 +176,7 @@ <h4 id="custom-elements-custom-tag-example">Creating a custom tag</h4>
 <p>We then need to use this class to define the element:</p>
 
 <pre class="highlight">
-document.defineElement("flag-icon", FlagIcon);</pre>
+document.registerElement("flag-icon", FlagIcon);</pre>
```

Comment:
Agree. But if you check window.customElements in browser then it isn't define yet. But, document.registerElement at least is present in Firefox and Chrome. For the shake of the example let's consider use registerElement. After all by using registerElement the example will work in some way.